### PR TITLE
Added global alert_context option

### PR DESCRIPTION
**Pull request by grega**
Allows:

```
  class { 'monit':
    alert_emails  => 'example@email.com',
    alert_context => 'but not on { pid, resource, instance }',
  }
```

which results in the following in `monitrc`:

```
 set alert example@email.com but not on { pid, resource, instance }
```

Ref: https://mmonit.com/monit/documentation/monit.html#Setting-an-event-filter

### CODE REVIEW

**Comment by grega** [comment]
Travis build failure doesn't appear to be related to changes in this PR:

```
Failures:

  1) monit On a RedHat OS with no package name specified should contain Package[monit]

     Failure/Error: should contain_package('monit')

     Puppet::PreformattedError:
       Evaluation Error: Error while evaluating a Function Call, 'versioncmp' parameter 'a' expects a String value, got Undef at /home/travis/build/echoes-tech/puppet-monit/spec/fixtures/modules/monit/manifests/params.pp:47:10 on node testing-worker-linux-docker-f341370a-3190-linux-4.prod.travis-ci.org
```

**Comment by dgarbus** [comment]
I would also like to see this merged. I am currently running this in my test environment without issue.

File: README.md [hunk 1]
```diff
@@ -119,6 +119,10 @@ The following parameters are available in the `::monit` class:
 
 Specifies one or more email addresses to send global alerts to. Valid options: array. Default value: []
 
+##### `alert_overrides`
```
Comment (FlorentPoinsaut) [inline README.md]: Can you name the variable alert_context please?

**Review by FlorentPoinsaut — CHANGES_REQUESTED** [review]
Hello,

Thank you for this PR.
I propose some changes.
Could you please update it?
Adding tests would be great :-)

Thank you.
Regards,

File: README.md (hunk 1 above)
Comment (grega) [reply README.md]: Done 👍

**Pull request closed without merging**